admin-dev/themes/new-theme/scss/components/layout/_content_div.scss

```diff
@@ -28,9 +28,20 @@
     padding-top: 0;
 
     .header-toolbar {
-      padding-left: 0;
-      margin: 0 -#{$grid-gutter-width / 2};
-      margin-bottom: 1rem;
+      display: none;
+    }
+  }
+
+  @at-root .multishop-enabled & {
+
+    &.-notoolbar {
+
+      .header-toolbar {
+        display: block;
```

Comment:
If the class has -notoolbar, then header-toolbar is not displayed => that sounds logical to me.
Why there is an exception for multistore ? or do I understand wrong ?

Reply:
By default `-notoolbar` means that header toolbar must be hidden. This class used for pages that implement their header-toolbar (e.g Stock page) or pages that doesn't require header-toolbar (e.g. Product page).

But multistore-toolbar will be inserted inside `-notoolbar`. 
So this exception must be handled.